src/Traits/CommonMutableContainerTrait.php

```diff
@@ -12,56 +12,67 @@
     use ExtractTrait;
 
     /**
-     * {@inheritdoc}
+     * @return ArrayList
```

Comment:
Why no **{@inheritdoc}** ? 
We need to show the docs for this method, which is on the Interface